Source/options.js

```diff
@@ -301,7 +315,9 @@ $(function() {
         patterns[0][3] = true;
         activatePattern(0, patterns);
       } else {
-        saveBulk({'savedPatterns': JSON.stringify(patterns)}, 'Deleted Pattern');
+        saveBulk({
+          'savedPatterns': JSON.stringify(patterns)
+        }, 'Deleted Pattern');
```

Comment:
The function call line is a little. So breaking it this way adds more readability.